docs/UserGuide.adoc

```diff
@@ -107,6 +107,11 @@ after a command has been input
 it adds money to the club's funds).
 * The amount input should be negative if the transaction is an expenditure made by the club (that is, it deducts money
 from the club's funds).
+
+[WARNING]
+Do not key in a quantity where the total amount in table or subtotal of that row exceeds 2,147,483,647, the range
+of an integer. If it exceeds this number, please key in a smaller quantity or checkout items separately. +
+You are advised to key in a number smaller than 10000 due to size limitation.
```

Comment:
Actually, I estimated 10000. Maybe you can try to key in different inputs on the actual app to see how much space it takes up?

Reply:
I think it's okay, just leave it as a ballpark figure. We can KIV this for next week?

Reply:
Also, since decimals are allowed in our amounts, maybe you can state "digits" instead of an integer?

Reply:
Yup sure, there is no need to rush!